numpy/core/machar.py

```diff
@@ -41,7 +41,8 @@ class MachAr:
         being no leading zeros in the mantissa.
     xmin : float
         Floating point number ``beta**minexp`` (the smallest [in
-        magnitude] usable floating value).
+        magnitude] usable floating value with there
+        being no leading zeros in the mantissa).
```

Comment:
Similar comment to my one above - rather than the "0's in the mantissa" detail, I'd prefer something like "with full precision" or "without additional loss of precision".